build/commands/lib/config.js

```diff
@@ -372,6 +372,15 @@ Config.prototype.enableCDMHostVerification = function () {
   return enable
 }
 
+Config.prototype.isDrumBrakeEnabled = function () {
```

Comment:
default values should go in gn so nothing should be added to config.js for this

Reply:
v8_enable_drumbrake override should go in build/args/brave_defaults.gni

Reply:
resolved